main.cc

```diff
@@ -1438,8 +1438,13 @@ int main(int ac, char** av) {
 
             static redis_service redis;
             if (cfg->redis_port() || cfg->redis_ssl_port()) {
-                with_scheduling_group(dbcfg.statement_scheduling_group, [proxy = std::ref(proxy), db = std::ref(db), auth_service = std::ref(auth_service), mm = std::ref(mm), cfg] {
-                    return redis.init(proxy, db, auth_service, mm, *cfg);
+                // Create an smp_service_group to be used for limiting the
+                // concurrency when forwarding Alternator request between
```

Comment:
Please either drop the comment altogether or fix up the mention of Alternator.

Reply:
Dropped the comment